src/main/java/pw/itr0/kaba/encrypt/PasswordUtil.java

```diff
@@ -4,9 +4,30 @@
 import java.util.Arrays;
 import java.util.Objects;
 
-public class PasswordUtil {
+/**
+ * Encryption utility for password of vaults, such as the password used to unlock the keystore.
+ * <p>
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [<p> tag should be placed immediately before the first word, with no space after.](https://www.codacy.com/app/ryotan/kaba-vault/file/2578430309/issues/source?bid=3298589&fileBranchId=3301306#l9)